backend/src/main/resources/db/migration/V15__populate_active_orchard_spu.sql

```diff
@@ -1,262 +1,262 @@
-         ('815', -1, 'True', 'False', 'True'),
-         ('816', 103, 'False', 'True', 'False'),
-         ('818', 103, 'False', 'True', 'False'),
-         ('819', 103, 'False', 'True', 'False'),
-         ('820', -1, 'True', 'False', 'True'),
-         ('821', 103, 'False', 'True', 'False'),
-         ('822', 68, 'False', 'True', 'False'),
-         ('823', 68, 'False', 'True', 'False'),
-         ('824', 103, 'True', 'False', 'False'),
-         ('825', 103, 'False', 'True', 'False'),
-         ('826', 103, 'False', 'True', 'False'),
-         ('829', 78, 'False', 'True', 'False'),
-         ('830', 104, 'True', 'False', 'False'),
-         ('831', 108, 'False', 'True', 'False'),
-         ('833', 78, 'False', 'True', 'False'),
-         ('834', -1, 'False', 'True', 'True'),
-         ('835', 7, 'False', 'True', 'False'),
-         ('836', 78, 'True', 'False', 'False'),
-         ('837', -1, 'False', 'True', 'True'),
-         ('838', 7, 'False', 'True', 'False'),
-         ('839', 68, 'False', 'True', 'False'),
-         ('840', 55, 'False', 'True', 'False'),
-         ('841', 103, 'False', 'True', 'False'),
-         ('842', 78, 'True', 'False', 'False'),
-         ('989', -1, 'False', 'True', 'True'),
-         ('990', -1, 'True', 'False', 'True'),
-         ('991', 7, 'False', 'True', 'False'),
-         ('992', 22, 'True', 'False', 'False'),
-         ('993', 4, 'True', 'False', 'False'),
-         ('994', -1, 'True', 'False', 'True'),
-         ('995', -1, 'True', 'False', 'True'),
-         ('996', 7, 'True', 'False', 'False'),
-         ('997', -1, 'True', 'False', 'True'),
-         ('998', 54, 'True', 'False', 'False'),
-         ('999', 85, 'True', 'False', 'False'),
-         ('110', 6, 'False', 'True', 'False'),
-         ('110', 7, 'True', 'True', 'False'),
-         ('144', 55, 'True', 'False', 'False'),
-         ('144', 108, 'True', 'False', 'False'),
-         ('163', 21, 'True', 'True', 'False'),
-         ('163', 22, 'True', 'True', 'False'),
-         ('206', 67, 'True', 'False', 'False'),
-         ('206', 68, 'True', 'False', 'False'),
-         ('214', 67, 'True', 'False', 'False'),
-         ('214', 68, 'True', 'False', 'False'),
-         ('217', 56, 'True', 'True', 'False'),
-         ('217', 57, 'True', 'True', 'False'),
-         ('230', 29, 'True', 'False', 'False'),
-         ('230', 43, 'True', 'False', 'False'),
-         ('235', 29, 'True', 'False', 'False'),
-         ('235', 35, 'True', 'False', 'False'),
-         ('235', 45, 'True', 'False', 'False'),
-         ('303', 73, 'True', 'True', 'False'),
-         ('303', 74, 'True', 'True', 'False'),
-         ('340', 38, 'True', 'False', 'False'),
-         ('340', 39, 'True', 'False', 'False'),
-         ('614', 67, 'True', 'False', 'True'),
-         ('614', 68, 'True', 'False', 'True'),
-         ('809', 103, 'True', 'True', 'False'),
-         ('809', 104, 'True', 'True', 'False');
+insert into active_orchard_spu
```

Comment:
It seems you forgot the `spar` schema . Good job, though!

Reply:
I did, but I added it later. This is an outdated version of the file, the current one is correct (otherwise the CI workflow would've failed).